jboss-maven/module.yaml

```diff
@@ -1,7 +1,85 @@
 schema_version: 1
 name: jboss-maven
 version: '1.0'
-description: Legacy jboss-maven script package.
+description: Maven module
+labels:
+    - name: io.fabric8.s2i.version.maven
+      value: "3.5"
```

Comment:
Do we need a more precise version?  This is truncated compared to the old version number we were setting.

Reply:
From what I understand, I don't think we need a more precise version

With or without this label, the images build and pass all tests; I believe it is there for cosmetic reasons. I brought this label over from the redhat-openjdk-18-openshift image: ( https://github.com/jboss-container-images/redhat-openjdk-18-openshift-image/blob/openjdk18-dev/image.yaml#L9-L10 ) and changed its value to match that of the environment variable, MAVEN_VERSION

If we were to define a more precise version, wouldn't we need to update it more frequently? Also, how do we know which version to choose? As I understand, concreate pulls the latest version of the defined package, rh-maven35, specified by the software collections repo, but we don't know the exact version specified for sure until the image is built. If we leave the label with just the major version, we will always be correct

Reply:
I can go along with that, but the same argument held true previously, so... @goldmann, what's your take?

Reply:
It should point only to major.minor IMHO. So exactly as it is proposed now.